docs/content/en/docs/pipeline-stages/deployers.md

```diff
@@ -23,7 +23,7 @@ Skaffold supports the following tools for deploying applications:
 The `deploy` section in the Skaffold configuration file, `skaffold.yaml`,
 controls how Skaffold builds artifacts. To use a specific tool for deploying
 artifacts, add the value representing the tool and options for using the tool
-to the `deploy` section.
+to the `deploy` section. You may even use different tools at the same time.
```

Comment:
```suggestion
to the `deploy` section. You may even use a combination of each available deployment approach at the same time.
```

Reply:
I'll not apply this suggestion right now, to not interfere with the kokoro run. But please go ahead and apply this if you think it's ready for merging!

Reply:
I just pushed it - will merge as soon as kokoro is done again :)